bin/rights_change.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Compare 2 hathifiles and report which items have changed rights
+# from ic/bib in file1 to pd/bib in file2.
+#
+# Invoke thusly:
+# $ bash rights_change.sh f1 f2
+# Results end up in ic_to_pd_YYYYMMDD.txt.
+# Each record in the output are presumed to have changed from ic
+# to pd between the generation of the 2 files.
+# Script starts at the bottom.
+
+run(){
+    f1=$1
+    f2=$2
+    echo "Started"
+    # First, simplify input hathifiles to the data we want.
+    # All the ic/bib records from f1 into one file...
+    cut_sort $f1 ic > cut_sort_ic.tsv
+    # And all the pd/bib records from f2 into another file.
+    cut_sort $f2 pd > cut_sort_pd.tsv
```

Comment:
I guess only to `pd` and not also to `pdus` because we're specifically worried about things incorrectly opened as fed docs, which would be pd?